packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -267,6 +352,13 @@ Static group authorization should perform as expected.`
                 resolver = this.resources.blankResolver(typeName, fieldName)
             }
             const authExpression = this.authorizationExpressionOnSingleObject(rules, 'ctx.source')
+            if (protectPrivateFields) {
```

Comment:
The modified tests suggest this isn't correct.
context.result.operation is set to query in setOperationExpression, but then context.source.operation is checked for query in operationCheckExpression, fails, and returns null.